cxx/gendb.cc

```diff
@@ -246,7 +288,12 @@ std::map<std::string, std::vector<size_t>> GenDB::get_domain_inds(
 }
 
 // Unincorporates the value of class_name.ref_field where the primary key
-// equals class_item.
+// equals class_item. For singleton references, recursively unincorporates
+// from the reference class as well.
+// TODO there is a known issue where an entity with a value that is perfectly
```

Comment:
Can you create a GitHub issue for this with more details?  In particular, I'm not sure what perfect correlation means here.

Reply:
I rethought this and I don't think it's actually an issue (if the entities don't share a DAG path, the "correlated" one won't change after the Gibbs step, even if it technically should have been unincorporated in intermediate states -- by "correlated" I just mean that Reference A takes value X iff Reference B takes value Y. I deleted the comment.